mobile/cmp/popover/Popover.scss

```diff
@@ -0,0 +1,27 @@
+.xh-popover {
+  &__content-wrapper {
+    position: fixed;
+    z-index: 21;
+    background-color: var(--xh-bg);
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0.3);
```

Comment:
Candidate for a CSS var - specifically, does this work equally well in light and dark modes?